pycnite/linetable.py

```diff
@@ -142,19 +142,11 @@ class PyCodeLocation:
     INFO_NONE = 15
 
 
-class LineTableReader311(LineTableReader):
-    """Read code.co_linetable for Python 3.11+"""
-
-    def __init__(self, code: types.CodeType311):
-        super().__init__(code)
-        self.linetable = code.co_linetable
-        self.start = code.co_firstlineno
-        self.end = self.start
-        self.line = self.start
-        self.end_pos = len(self.linetable)
+class VarintReader:
+    """Mixin to read and parse packed varints in 3.11+."""
 
     def _read_byte(self):
-        b = self.linetable[self.pos]
+        b = self.table[self.pos]
```

Comment:
The type-check step is emitting attribute errors for `table` in this line and `pos` in the next. Declaring the types of the attributes via variable annotations should fix that.

Reply:
done